### PR TITLE
SignupPage is implemented

### PR DESCRIPTION
**Pull request by metekurt**

### CODE REVIEW

File: practice-app/react-front-end/src/views/SignupPage/SignupPage.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                            </InputAdornment>
+                          )
+                        }}
+                        
+                      />*/}
+                    
+                    
+                      
+                    
+          <FormControl className={classes.formControl}>
```
Comment (ufukyilmaz) [inline practice-app/react-front-end/src/views/SignupPage/SignupPage.js]: When a user type is selected from the user type selection menu, the selected value is not seen in the form.
(I tried it on Firefox and Chrome.)

File: practice-app/react-front-end/src/views/SignupPage/SignupPage.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+      setCardAnimation("");
+    }, 700);
+    const classes = useStyles();
+    const { ...rest } = props;
+    return (
+      <div>
+        <Header
+          absolute
+          color="transparent"
+          brand="Traders Platform"
```
Comment (ufukyilmaz) [inline practice-app/react-front-end/src/views/SignupPage/SignupPage.js]: Brand name can be changed as Khaji-it Traders Platform

Comment (metekurt) [reply practice-app/react-front-end/src/views/SignupPage/SignupPage.js]: İt is changed.

File: practice-app/react-front-end/src/views/SignupPage/SignupPage.js (hunk 1 above)
Comment (metekurt) [reply practice-app/react-front-end/src/views/SignupPage/SignupPage.js]: I fixed it.

**Comment by metekurt** [comment]
Some corrections have been made, can you check again?

**Pull request merged**